### PR TITLE
[1.12.x] docs cherry picks

### PR DESCRIPTION
**Pull request by thaJeztah**
cherry-picks of these PR's;

https://github.com/docker/docker/pull/26857 (https://github.com/docker/docker/pull/26857/commits/d69fce79f94b807ccb0f2f64c47881ee8fc31250)
https://github.com/docker/docker/pull/27059 (https://github.com/docker/docker/pull/27059/commits/c326e0afe39d19a789f0819cb3d81259167a762f)
https://github.com/docker/docker/pull/27341 (https://github.com/docker/docker/pull/27341/commits/72dd684c88976b7c76cceeafe0ab6c98c8963b62)
https://github.com/docker/docker/pull/27363 (https://github.com/docker/docker/pull/27363/commits/970ab9a224658e17894a7673189ad9d2491a592d)
https://github.com/docker/docker/pull/27428 (https://github.com/docker/docker/pull/27428/commits/441c9411e369f5146783c46d7a9413917dacaebe)
https://github.com/docker/docker/pull/27429 (https://github.com/docker/docker/pull/27429/commits/f988a9ce5e5926b211c7e762b67cff8e9dbac380)
https://github.com/docker/docker/pull/27431 (https://github.com/docker/docker/pull/27431/commits/8e02b29d247379d66280babcf654e467989bd4bb)
https://github.com/docker/docker/pull/27434 (https://github.com/docker/docker/pull/27434/commits/9ccb1f159e964384df32c083058197340941fafa https://github.com/docker/docker/pull/27434/commits/08fe24c2d5ef415d2b94f8e3f28d0299db2d7b6a)
https://github.com/docker/docker/pull/27526 (https://github.com/docker/docker/pull/27526/commits/2c288e60db1ecd47842e92432fbe0ab367033374)
https://github.com/docker/docker/pull/27546 (https://github.com/docker/docker/pull/27546/commits/98fb1644c2b9bf7d808a3d75de5997f87b9c60e6)
https://github.com/docker/docker/pull/27593 (https://github.com/docker/docker/pull/27593/commits/b39e65e9378cf6a32bcc3ac3bd8b982d339ea25d)
https://github.com/docker/docker/pull/27640 (https://github.com/docker/docker/pull/27640/commits/587b186f85383f98f5475b8c16b883553299b525 https://github.com/docker/docker/pull/27640/commits/2e0d6e6be2294ef2561e8c3b5f750fd814536268)
https://github.com/docker/docker/pull/27764 (https://github.com/docker/docker/pull/27764/commits/a5b69faafba8bbb85094993edf48bab106e7b54d)
https://github.com/docker/docker/pull/28451 (https://github.com/docker/docker/pull/28451/commits/2c38d43eb5276a82f8e076058506468e0d5a2f9f)
https://github.com/docker/docker/pull/28602 (https://github.com/docker/docker/pull/28602/commits/16c9cdad6fd7c78600e795cad03ec8b34cba0371)
https://github.com/docker/docker/pull/28607 (https://github.com/docker/docker/pull/28607/commits/db3f06fd976d71a1d335278ecedf026aea6046c0)
https://github.com/docker/docker/pull/28778 (https://github.com/docker/docker/pull/28778/commits/b9e65f40cfce3eb1907ad9d8a358c6ae1ad64175)
https://github.com/docker/docker/pull/28824 (https://github.com/docker/docker/pull/28824/commits/fc7c235f84f76d5d64f25bc802486ecbe87f8b95)
https://github.com/docker/docker/pull/28890 (https://github.com/docker/docker/pull/28890/commits/2333657e1b20d6a482627b78c9333cdf34397ff5)
https://github.com/docker/docker/pull/28906 (https://github.com/docker/docker/pull/28906/commits/2178c840ba420d7d207aedae292e2e70d7198265)
https://github.com/docker/docker/pull/28928 (https://github.com/docker/docker/pull/28928/commits/71f1205269494430116e7a564ee76bfa0bfccc63)
https://github.com/docker/docker/pull/29106 (https://github.com/docker/docker/pull/29106/commits/35c82f422d5db64baec6739490900f7f8fa7d7fd)
https://github.com/docker/docker/pull/29308 (https://github.com/docker/docker/pull/29308/commits/bac792c1634724def05c5ebc93891ecd5b353a53)

ping @mstanleyjones  - do not merge yet, as I think @andrewhsu is building binaries from this branch

### CODE REVIEW

File: docs/extend/index.md [hunk 1]
```diff
@@ -1,48 +1,50 @@
 ---
-redirect_from:
-- "/engine/extend/"
-title: "Docker Engine managed plugin system"
-description: "How develop and use a plugin with the managed plugin system"
-keywords: "API, Usage, plugins, documentation, developer"
-advisory: "experimental"
+advisory: experimental
```
Comment (mstanleyjones) [inline docs/extend/index.md]: Is this right? Going from not-experimental to experimental?

**Review by mdlinville — APPROVED** [review]
One question, then LGTM

File: docs/extend/index.md (hunk 1 above)
Comment (mstanleyjones) [inline docs/extend/index.md]: You're right. LGTM!

**Comment by thaJeztah** [comment]
# DON'T MERGE YET

1.12.x builds are currently running, so wait until the release is built/tagged

**Comment by vdemeester** [comment]
`1.12.4` is now released 👼 
Merging 👼

**Pull request merged**